CQUI.modinfo

```diff
@@ -69,12 +69,27 @@
         <UpdateDatabase id="CQUI_Settings_Local">
             <Properties>
                 <Name>CQUI Settings Local</Name>
-                <LoadOrder>-10</LoadOrder>
+                <LoadOrder>-11</LoadOrder>
             </Properties>
             <Items>
                 <File>Assets/cqui_settings_local.sql</File>
             </Items>
         </UpdateDatabase>
+        <!-- This is a "hack", however it is one that allows users to skirt the issue of their settings_local file being updated every time we push new updates -->
+        <!-- We are allowed to reference files in the parent folder to the .modinfo file, if the file exists then the game will load it up just fine. -->
+        <!-- However, these files in the parent folder are *not* uploaded to the Steam Workshop and do *not* appear in the [SteamPath]\SteamApps\Workshop\Content\289070 folder (the Civ6 Workshop Mods folder) of users that subscribe -->
```

Comment:
note - long lines make it harder to review changes (need to scroll back and forth). 
I would avoid lines longer than whatever fits on a typical screen, including the Github web UI "PR review" page.